<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreJeu</class>
 <widget class="QWidget" name="FenetreJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenetreJeu</string>
  </property>
  <widget class="QPushButton" name="pushButtonNouvelleZone">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNouvelleZone">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>195</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Port :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>5555</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelAdresse">
          <property name="text">
           <string>Adresse Ip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAdresse">
          <property name="text">
           <string>172.18.58.90</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonConnexionServeur">
      <property name="text">
       <string>Connexion serveur</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FenetreJeu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
